webapp/components/textbox.jsx

```diff
@@ -92,12 +82,12 @@ export default class Textbox extends React.Component {
         }
     }
 
-    handleChange(e) {
```

Comment:
I see this syntax being corrected throughout this file. What's the difference between the two? Are they functionally equivalent? Or is this more than a stylistic change?